rust-book-old/src/toolchain/katana/interact.md

```diff
@@ -0,0 +1,321 @@
+### install starkli
+
+```console
+curl https://get.starkli.sh | sh
+## open a new terminal
+starkliup
+```
+
+You can check your installation by running `starkli --version`,then you will get the starkli version.
+
+### install katana
+
+```console
+git clone https://github.com/dojoengine/dojo
+cd dojo
+cargo install --path ./crates/katana --locked --force
+```
+
+You can check your installation by running `katana --version`,then you will get the katana version.
+
+### Deploy and interact with contracts
+
+#### run katana node
+
+```console
+katana --accounts 3 --seed 0 --gas-price 250
+```
+
+After starting the node, the accounts you need will be automatically generated and deployed. We need to import the pre-deployed accounts into starkli for use.
+
+#### Configure starkli account
+
+1. Configure the signer, execute the following command, and then enter the private key. This private key is the private key of the account generated by kanata.
+
+```console
+starkli signer keystore from-key ~/.starkli-wallets/deployer/account0_keystore.json
+```
+
+2. Configure account description and create a new json file
+
+```console
+touch ~/.starkli-wallets/deployer/account0_account.json
+```
+
+3. Copy the following content into the json file
+
+```console
+{
+  "version": 1,
+  "variant": {
+        "type": "open_zeppelin",
+        "version": 1,
+        "public_key": "<SMART_WALLET_PUBLIC_KEY>"
```

Comment:
I'm not sure `SMART_WALLET_PUBLIC_KEY` is the right term here.
`ACCOUNT_PUBLIC_KEY` or `SIGNER_PUBLIC_KEY` should be better.